src/v1/database/models/contactIndex.js

```diff
@@ -0,0 +1,65 @@
+const dataPool = require('../config/config')
+
+const getAllContactsDB = (req, res) => {
+    dataPool.query('SELECT * FROM FANOUT_CONTACTS ORDER BY id ASC', (error, results) => {
+        if (error) {
+          throw error
+        }
+        res.status(200).json(results.rows)
```

Comment:
From what I understand, the data is typically pulled the database methods so that it can then be manipulated for business purposes. 

- Such as multiplying by 5 for whatever reason and so on.

Reply:
I think we need to clean up some of the functions. We're sending http responses from two places; the controller layer, and the db layer.